packages/clay-navigation/src/BreadcrumbItem.tsx

```diff
@@ -0,0 +1,98 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import classNames from 'classnames';
+import ClayButton from '@clayui/button';
+import React from 'react';
+
+export interface IBreadcrumbItem {
+	/**
+	 * Flag to indicate if the Breadcrumb item is active or not.
+	 */
+	active?: boolean;
+
+	/**
+	 * This value is used to be the target of the link.
+	 */
+	href?: string;
+
+	/**
+	 * Label of the Breadcrumb item
+	 */
+	label: string;
+
+	/**
+	 * Callback for when a Breadcrumb item is clicked.
+	 */
+	onItemClick?: (event: React.SyntheticEvent) => void;
```

Comment:
Since its already called a `BreadcrumbItem`, can we just call this `onClick` to reduce name redundancy?

Reply:
Yes, makes sense.